src/test/java/io/github/ygojson/application/yugipedia/client/AbstractYugipediaClientTest.java

```diff
@@ -66,7 +72,7 @@ private Response<QueryResponse> doExecuteTestQueryRecentChanges(
 		);
 	}
 
-	private Response<QueryResponse> doExecuteQueryPagesByTitle(
+	public Response<QueryResponse> doExecuteQueryPagesByTitle(
```

Comment:
Should stay private:
```suggestion
	private Response<QueryResponse> doExecuteQueryPagesByTitle(
```